xds/src/main/java/io/grpc/xds/XdsNameResolver.java

```diff
@@ -806,25 +806,25 @@ private void cleanUpRoutes() {
     }
 
     private void cleanUpRouteConfigState() {
```

Comment:
nit: method can be further renamed.

Reply:
Oops, missed this one. Should be fixed now.